documentation/source/contribute/contribute_openebs_doc.rst

```diff
@@ -49,7 +49,7 @@ Creating an Issue
 Creating a Branch
 -----------------
 
-1. Go to the master on Github and create a branch using the following command.
+1. Go to the openebs fork (https://github.com/openebs/openebs) on Github and create a branch on the openebs fork using the following command.
```

Comment:
Let split this into two.. 

1. First, create your openebs fork from (https://github.com/openebs/openebs). If you already have a forked openebs, rebase with the master to get the latest changes. 
2. Create a branch on your openebs fork using the following command.

Reply:
Done!